qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import static org.opensearch.knn.TestUtils.KNN_BWC_PREFIX;
+
+public class IndexingIT extends AbstractRestartUpgradeTestCase {
+    private final String CLASS_NAME = "indexingit";
+    private final String testIndexName = KNN_BWC_PREFIX + CLASS_NAME + "test-index";
```

Comment:
I think we need to follow existing name convention and make constant names uppercase